test/v1x/cpp/example/cpp_standalone.cc

```diff
@@ -0,0 +1,31 @@
+#include <openql>
+
+#include <iostream>
+
+
+int main(int argc, char **argv) {
```

Comment:
I think this could be a google test as the others.
It's an example but can totally be a test in the big test executable.

Reply:
Will also save on link time